CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## 0.4.0-alpha
+
+This code is considered production quality, but depends on angular2: 3.0.0-alpha.
+The alpha tag represents the evolving nature of the AngularDart api, not
+code quality (3.0.0-alpha is used in production Google apps).
+
+Component update rollup.
```

Comment:
Not at all. It is just an artifact of how I was tracking changes. I will remove.